src/styles/components/framework-configuration/styles.less

```diff
@@ -37,3 +37,12 @@
 .glyphicon-plus {
   font-style: normal;
 }
+
+.outdated-package-warning-оutline {
+  border: @base-spacing-unit * 1/12 solid @warning-color;
+  border-bottom-left-radius: 5% 20%;
```

Comment:
This will fail the build, since it expects us to use the shorthand property. But the shorthand property `5% 20%` does not yield the same result. And if we use `5% / 20%`, it's interpreted as `4%` probably because we use .less.

Reply:
@mperrotti 	we might want to bring this pattern to `ui-kit`.

it looks like `outline` is not an `appearance` but rather an independent prop that changes the look of a badge different from the usual appearances.

<img width="815" alt="63838781-4740c480-c97e-11e9-9f40-dec13423270b" src="https://user-images.githubusercontent.com/300861/63868821-73c50280-c9b7-11e9-90b4-5022280016d5.png">

Reply:
It's from Izhar.